lib/sidekiq/cron/poller.rb

```diff
@@ -5,20 +5,17 @@
 
 module Sidekiq
   module Cron
-    POLL_INTERVAL = Sidekiq::Options[:average_scheduled_poll_interval] || 30
+    POLL_INTERVAL = 30
 
     # The Poller checks Redis every N seconds for sheduled cron jobs.
     class Poller < Sidekiq::Scheduled::Poller
-      def initialize
-        Sidekiq.configure_server do
-          Sidekiq::Options[:poll_interval_average] = POLL_INTERVAL
-        end
-
+      def initialize(options)
         if Gem::Version.new(Sidekiq::VERSION) >= Gem::Version.new('6.5.0')
-          # Sidekiq Poller init requires a config argument.
-          super(Sidekiq)
-        else
           super
+        else
+          # Old version of Sidekiq does not accept a config argument.
+          @config = Sidekiq.options
+          super()
```

Comment:
Newer versions of Sidekiq pass in the options to the poller so that we don't have to read the global config. I think this is for the future where you can run multiple Sidekiq instances side-by-side.

So instead of making this behave like the old poller, we should follow this new strategy and just fall back to reading the global config on older versions.